content/en/docs/concepts/workloads/controllers/statefulset.md

```diff
@@ -386,7 +386,7 @@ StatefulSet will then begin to recreate the Pods using the reverted template.
 
 ## PersistentVolumeClaim retention
 
-{{< feature-state for_k8s_version="v1.27" state="beta" >}}
+{{< feature-state for_k8s_version="v1.31" state="stable" >}}
```

Comment:
`{{< feature-state feature_state_name="StatefuleSetAutoDeletePVC" >}}`

Reply:
Done. Further up in the document is the tag feature_gate_name, is that what I should use?

Reply:
I think so. The shortcode can/should be removed when the gate is removed.

Reply:
Oh ..., you may want to remove line 390.

Reply:
done